app/models/shipping.rb

```diff
@@ -0,0 +1,76 @@
+  require 'active_shipping'
+
+  # returns shipping rates and determines the size of a package given weight
+  # uses active shipping to calculate and return rates
+  class Shipping
+
+    SMALL = [5, 5, 5]
+    MEDIUM = [10, 10, 10]
+    LARGE = [20, 20, 20]
+
```

Comment:
This is a really smart way to simplify and handle different package sizes for this project!